deploy/Chart/templates/database/pv.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-sts-0
+  namespace: "{{ .Release.Namespace }}"
+  labels:
+    control-plane: database
+    app.kubernetes.io/name: database
+    app.kubernetes.io/part-of: radius
+spec:
+  storageClassName: standard
+  persistentVolumeReclaimPolicy: Retain
+  capacity:
+    storage: 3Gi
+  accessModes:
+    - ReadWriteMany
```

Comment:
Is there a reason why we need this? Should be ReadWriteOnce unless we need otherwise.